picard/util/time.py

```diff
@@ -1,49 +1,49 @@
-# -*- coding: utf-8 -*-

```

Comment:
I committed the line ending fixes separately in https://github.com/metabrainz/picard/commit/28c6118bedfefff11f1b3defbb56d753c8aed5a9